src/main/java/eu/uftplib/service/UftpSigningServiceImplementation.java

```diff
@@ -27,21 +27,18 @@
 public class UftpSigningServiceImplementation implements UftpSigningService {
 
     private String role;
+    private LazySodiumJava lazySodium;
 
     public UftpSigningServiceImplementation(String role) {
         this.role = role;
+        this.lazySodium = new LazySodiumJava(new SodiumJava(), new Base64MessageEncoder());
```

Comment:
The import for Base64MessageEncoder is missing. Please make sure the code compiles

Reply:
Correction pushed to branch on appbeats repo (https://github.com/appbeats/uftplib/tree/base64fix).

Could you give my account write access on this branch please?

Reply:
Changes now pushed to this branch.